recipes/r-stitch/meta.yaml

```diff
@@ -1,17 +1,24 @@
-{% set version = "1.6.8" %}
-{% set sha256 = "a1b7c3fe273f2f2dee0bfb34f0afc95aeab7a92333d3bc15679abbca6dc553f8" %}
+{% set version = "1.6.10" %}
+{% set sha256_package = "1e432655e299bb5941104c2645ed3a6b3c2c4a44b583c5aaa2b20d2538875f9e" %}
+{% set sha256_script = "0c61d79cf519e5c5ae33807725577da3d1599bcc89413c3a6bce3fb7c99090c7" %}
 
 package:
   name: r-stitch
   version: '{{ version }}'
 
 source:
-  url: https://github.com/rwdavies/stitch/releases/download/{{ version }}/STITCH_{{ version }}.tar.gz
-  sha256: '{{ sha256 }}'
+  - url: https://github.com/rwdavies/stitch/releases/download/{{ version }}/STITCH_{{ version }}.tar.gz
+    sha256: '{{ sha256_package }}'
+  - url: https://raw.githubusercontent.com/rwdavies/STITCH/{{ version }}/STITCH.R
+    sha256: '{{ sha256_script }}'
 
 build:
-  number: 2
-  script: '$R CMD INSTALL --build --install-tests .'
+  number: 0
+  script: 
+    - 'mkdir -p $PREFIX/bin && cp STITCH.R $PREFIX/bin'
+    - '$R CMD INSTALL --build --install-tests .'
+  script_env:
+    - 'LC_ALL=en_US.UTF-8'
```

Comment:
Is this really needed?

Reply:
Does this influence the user-setup?

Reply:
My build was failing locally otherwise for missing locales using docker

Reply:
Mh I can try to remove and see if it builds here on Azure.

Reply:
Mh it was failing the test without when using bioconda-utils build --docker but interestingly it was working with bioconda-utils build without the docker flag. I can try to see if it builds here on Azure!

Reply:
Seems to work! I think then I could have been because my local system was on a en_GB.UTF-8 locale which is set with -e LANG=en_GB.UTF-8 in the container run command, and this locale is not available inside the container.